4.0/Dockerfile

```diff
@@ -49,6 +49,9 @@ ENV PASSENGER_CONTAINER_SCRIPTS_PATH=/usr/share/container-scripts/passenger/ \
 
 RUN /usr/libexec/passenger-prepare
 
+# Reset permissions of filesystem to default values
+RUN rpm-file-permissions
```

Comment:
`/usr/libexec/passenger-prepare` contains other configuration code to prepare image.